ethereum/contracts/pyth/PythState.sol

```diff
@@ -21,6 +21,8 @@ contract PythStorage {
         // (chainId, emitterAddress) => isValid; takes advantage of
         // constant-time mapping lookup for VM verification
         mapping(bytes32 => bool) isValidDataSource;
+
+        uint singleUpdateFeeInWei;
```

Comment:
Do we want to add `singleUpdateFeeInWei` to `Pyth.sol`'s `initialize` function, so that when we deploy on new chains we can initialize it atomically with the other state variables? I don't think it's strictly necessary but it's nice to treat all the state variables equally.

Reply:
This way is actually contradicts with truffle migration philosophy and can cause inconsistent updates. We are not setting any value here, but if any value should be set, it should be set during the same migration step that the deployed one will go for. 

Once we go for governance, the sequence of governance messages will do that.